aws_cloudtrail_rules/aws_resource_made_public.py

```diff
@@ -66,4 +66,15 @@ def rule(event):
     if policy == "":
         return False
 
-    return policy_is_not_acceptable(json.loads(policy))
+    return False
```

Comment:
Does this not need this check?

Reply:
Oh wow, good catch. That was just a bad git merge on my part. Pushing an update shortly.